### ISSUE TITLE
"cannot perform relative import" with namespace packages

### ISSUE THREAD

**Issue by lilydjwg** [python/mypy#9307]
* this is a bug with [namespace packages](https://www.python.org/dev/peps/pep-0420/).

* code

```sh
mkdir t
echo 'FOO = 1' > t/a.py
echo 'from . import a' > t/b.py
mypy t
```

* What is the actual behavior/output?

```
t/b.py:1: error: No parent module -- cannot perform relative import
Found 1 error in 1 file (checked 2 source files)
```

* What is the behavior/output you expect?
```
Success: no issues found in 2 source files
```

* What are the versions of mypy and Python you are using?

mypy 0.782 and Python 3.8.5

* Do you see the same issue after installing mypy from Git master?

yes

* What are the mypy flags you are using? (For example --strict-optional)

None.

**Comment by gvanrossum** [comment]
When you pass a directory to mypy, it type-checks the files in the directory, assuming the directory is *not* a package. To treat the directory as a package, you would need to use `mypy -p t`. Unfortunately there's still a bug there and the workaround is to add an empty `__init__.pyi` to the directory.

Closing as duplicate of #5759.
